tests/unit/mainwindow/test_tabwidget.py

```diff
@@ -128,6 +129,22 @@ def test_update_tab_titles_benchmark(self, benchmark, widget,
 
         benchmark(widget.update_tab_titles)
 
+    def test_tab_min_width(self, widget, fake_web_tab, config_stub):
+        """Ensure by default, all tab sizes are the same."""
+        widget.addTab(fake_web_tab(), 'foobar')
+        normal_size = widget.tabBar().minimumTabSizeHint(0).width()
+        normal_size += 100
+        config_stub.val.tabs.min_width = normal_size
+        assert widget.tabBar().minimumTabSizeHint(0).width() == normal_size
```

Comment:
Wouldn't it make more sense to look at the size the tab actually got, via `tabRect(0)`?